hathor/transaction/storage/transaction_storage.py

```diff
@@ -624,6 +624,11 @@ def get_best_block_tips(self, timestamp: Optional[float] = None, *, skip_cache:
             elif meta.score > best_score:
                 best_score = meta.score
                 best_tip_blocks = [block_hash]
+
+        # XXX: if there's more than one we filter it so it's the smallest hash
+        if len(best_tip_blocks) > 1:
```

Comment:
#540 sort of does this by using the height-index, but because it's not using this PR as a base it does this incorrectly and hides other tips, after this PR that change would be correct